### PR TITLE
chore: bring back json schemas to avoid a breaking change in schemastore and other third parties

### LINKED ISSUE #4149: schema.json files were removed
Hello everyone! I saw in this [PR](https://github.com/OAI/OpenAPI-Specification/pull/4137) that the files https://raw.githubusercontent.com/OAI/OpenAPI-Specification/main/schemas/v3.1/schema.json and https://raw.githubusercontent.com/OAI/OpenAPI-Specification/main/schemas/v3.0/schema.json were removed because they should not be the source of truth. It turns out that I use these files a lot in my CI to compare and validate my OpenAPI schemas in many tests. I download and store them, but I update the file every 1 hour and now it's failing and breaking my CI.

I generate my OpenAPI schemas in JSON and not in YAML, which is a practice used by many and accepted by the specification.
Is there any chance to revert this change? 

Thanks.

### PR DESCRIPTION
**Pull request by ThomasRooney**
Closes #4149 

Reverts a section of #4137 for reasons stated in #4149

### CODE REVIEW

File: schemas/v3.0/schema.json
Comment (ralfhandl) [inline schemas/v3.0/schema.json:1]: This file was intentionally deleted, please use this URL
* https://spec.openapis.org/oas/3.0/schema/2021-09-28

File: schemas/v3.1/dialect/base.schema.json
Comment (ralfhandl) [inline schemas/v3.1/dialect/base.schema.json:1]: This file was intentionally deleted, please use this URL
* https://spec.openapis.org/oas/3.1/dialect/base

File: schemas/v3.1/meta/base.schema.json
Comment (ralfhandl) [inline schemas/v3.1/meta/base.schema.json:1]: This file was intentionally deleted, please use this URL
* https://spec.openapis.org/oas/3.1/meta/base

**Review by ralfhandl — CHANGES_REQUESTED** [review]
The schemas are all available at the same location as their `$id` or `id` says.

They were intentionally removed.

**Review by lornajane — APPROVED** [review]
I'm in favour of temporarily accepting this change and communicating with the projects that experienced issues about reinstating it.

**Comment by ThomasRooney** [comment]
Schemastore, when updated, should automatically fix everything within a few days as the TTL is relatively low. I'm in favour of closing this PR.

**Comment by ThomasRooney** [comment]
Closing this based on https://github.com/SchemaStore/schemastore/pull/4159 fixing this in a safer way.

**Pull request closed without merging**